powersimdata/data_access/context.py

```diff
@@ -18,4 +22,8 @@ def get_data_access(data_loc=None):
             root = server_setup.BACKUP_DATA_ROOT_DIR
         else:
             root = server_setup.DATA_ROOT_DIR
-        return SSHDataAccess(root)
+
+        mode = get_deployment_mode()
+        if mode == DeploymentMode.Server:
+            return SSHDataAccess(root)
+        return LocalDataAccess(root)
```

Comment:
Just being curious, what is the advantage of having a class with 2 static attributes over a dictionary (`deployment_mode = {"Server": "SERVER", "Container": "CONTAINER"}`)?

Reply:
Not sure there is an advantage either way. I think the class works better with tab completion and seems slightly more readable. Another option would be an enum.